apps/mobile/src/utils/analytics.ts

```diff
@@ -13,3 +14,24 @@ export const analytics = configureAnalyticsClient<SegmentClient>({
     platform: 'mobile',
   },
 });
+export function useAnalytics() {
```

Comment:
@camerow I think we need to respect this setting. Tagging you as an FYI as I look at it 

See https://github.com/leather-io/mono/pull/666#issuecomment-2527590291